taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -276,6 +277,45 @@ export const inject = (deps: InjectedDependencies) => {
 		vscode.window.showInformationMessage(msg)
 			.then(_ => Promise.resolve()) as Promise<void>;
 
+	const logAllNestedErrors = (err: TaqVsxError | TaqError | Error | any, output: Output) => {
```

Comment:
Is it possible to avoid the `any` here please? 

In general, I'd like to avoid using the `any` type in the codebase.

Reply:
It is, but then we have to know all the umbrella types that cover all possible errors. In a language like java, with propagated errors, you know it from the function declaration. In js/ts, you don't know that.

Do you have any workarounds on your mind?